libpod/options.go

```diff
@@ -946,6 +946,17 @@ func WithHosts(hosts []string) CtrCreateOption {
 	}
 }
 
+// WithHostname sets additional hostname for the hosts file
+func WithHostname(hostname string) CtrCreateOption {
```

Comment:
No need for this - we expect that the hostname will be set in the spec when it's passed in